src/pages/RequestSchoolProducts/RequestSchoolProducts.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, useState } from 'react';
+import styles from './RequestSchoolProducts.module.scss';
+import RadioGroup from '@/components/RadioGroup/RadioGroup';
+import TextInput from '@/components/TextInput/TextInput';
+import TextArea from '@/components/TextArea/TextArea';
+import FormButton from '@/components/FormButton/FormButton';
+
+const RequestSchoolProducts: FC = () => {
+  const [name, setName] = useState<string>();
+  const [email, setEmail] = useState<string>();
+  const [phone, setPhone] = useState<string>();
+  const radioButtonLabels = [
+    'I am a parent or guardian',
+    'I work at another school',
+    'I work for a charity or volunteer group',
+    'Something else',
+  ];
+  const radioButtonValues = [
+    'parentGuardian',
+    'anotherSchool',
+    'charityVolunteerGroup',
+    'somethingElse',
+  ];
+  const onNameChange = (value: string): void => {
+    setName(value);
+  };
+  const onEmailChange = (value: string): void => {
+    setEmail(value);
+  };
+  const onPhoneChange = (value: string): void => {
+    setPhone(value);
+  };
+  return (
```

Comment:
Can we add some more whitespace/new lines on this page please? Especially around the return and the onChange handlers. Makes it easier to read.